modules/rules_proto_grpc/5.0.0-alpha2/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(
+    name = "rules_proto_grpc",
+    version = "5.0.0-alpha2",
```

Comment:
Would it be problematic if version resolution results in `rules_proto_grpc@5.5.0` but e.g. `rules_proto_grpc_buf@5.0.0`? That is, does the main module version have to match the submodules exactly or is it sufficient if it is at least as recent? Depending on the answer to that question, you may have to add `bazel_dep`s on all the submodules here to force the versions to be identical.

Reply:
Hmm this is a tough one. In theory they should be fine to skew versions and I'm trying to think of a scenario where a skewed core version breaks a language module (assuming following good practices like not loading from internal bzl files).

On the other hand, it'd be nice to lock the versions from the perspective of reducing unknowns that aren't tested, but I'm somewhat hesitant to put the reverse deps in. This is because then pulling any one language then pulls core which then pulls every other language, which _somewhat_ loses the benefit of isolating each language to its own module (ok they wouldn't actual be a big deal if not referenced in the workspace).

I think for the time being I just allow the versions to skew and if it bites me then I have to come back and add the reverse deps later.